core/collateral/engine.go

```diff
@@ -734,7 +734,39 @@ func (e *Engine) FinalSettlement(ctx context.Context, marketID string, transfers
 	}
 
 	if !settle.Balance.IsZero() {
-		e.log.Panic("settlement balance is not zero", logging.BigUint("balance", settle.Balance))
+		e.log.Error("final settlement left asset unit in the settlement, transferring to the asset reward account", logging.String("remaining-settle-balance", settle.Balance.String()))
```

Comment:
Does "asset unit" imply it's just `1`? Because at this point it's not clear that's the case (maybe more). Maybe I am just misreading.

Reply:
it is the case in your log, I don't know if it's the general case - need to reproduce it to get the root cause. Happy to make it any amount you think makes sense.